QuasistaticBrownianThermalNoise/QuasistaticBrownianThermalNoise.cpp

```diff
@@ -500,8 +540,31 @@ CylTransFunc::CylTransFunc(double coatThick, double halfCylThick):
     //kIso_AlGaAs #effective isotropic approx to x=0.92 AlGaAs
     //kIso_Ta2O5 #tantalum
     //kIso_FusedSilica #fused silica
-    mWhichCoatingYijkl = kAlGaAs;
+    mWhichCoatingYijkl =   kIso_Ta2O5;
     mWhichSubstrateYijkl = kIso_FusedSilica;
+      
+      
+    lossPhi_Iso_FusedSilica = 1.e-6; //Cole+ (2013)
+    //    lossPhi_Iso_Ta2O5 = 4.e-4; //Yamamoto+ (2006), PRD 74, 022002
+    // (good value??)
+    lossPhi_Iso_Ta2O5 = 2.3e-4; // Value from Hong + et. al.
+
+    lossPhi_Iso_AlGaAs = 2.5e-5; //Cole+ (2013)
+    lossPhi_AlGaAs = lossPhi_Iso_AlGaAs; //use same loss angle from Cole+(2013)
+      
+    phi11 = 2.5e-5;
+    phi12 = 2.5e-5;
+    phi44 = 2.5e-5;
+    
+      
+    lossPhi_Iso_FusedSilica_bulk  = lossPhi_Iso_FusedSilica;
+    lossPhi_Iso_FusedSilica_shear = lossPhi_Iso_FusedSilica;
+    
```

Comment:
Shouldn't this have different values for bulk and shear as well? Do such values exist in the literature? If you just can't find them, make a note in a comment that we have no experimental values yet for these loss angles, so we make them equal.